app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,7 @@
 <resources>
     <string name="app_name">AndroidAcademyApp</string>
+    <string name="edit_profile">Edit Profile</string>
```

Comment:
Let's add russian translation as well, please

Reply:
imho: we should decide do it right when you implement some new feature or use some single language for now and translate all after